.eslintrc.json

```diff
@@ -50,8 +50,22 @@
     "space-before-blocks": 2,
     "strict": [2, "global"],
     "vars-on-top": 2,
-    "yoda": [2, "never"]
+    "yoda": [2, "never"],
+    "sort-keys": ["error", "asc"]
   },
+  "overrides": [
+    {
+      "files": [
+        "test/**/*.js",
+        "features/**/*.js",
+        "features-sdk/**/*.js",
+        "lib/api/controllers/memoryStorage.js"
```

Comment:
Memory storage controller use object to declare it's methods, so the property order is important